synapse/cortex.py

```diff
@@ -410,6 +399,32 @@ def finiCortex():
 
         self.onfini(finiCortex)
 
+    def _initCoreLayers(self):
+        import synapse.cells as s_cells  # avoid import cycle
+        layersdir = pathlib.Path(self.dirn, 'layers')
+        layersdir.mkdir(exist_ok=True)
+        if pathlib.Path(layersdir, 'default').is_dir():
+            self._migrateOldDefaultLayer()
+
+        # Layers are imported in reverse lexicographic order, where the earliest in the alphabet is the 'topmost'
+        # write layer.
+        for layerdir in sorted((d for d in layersdir.iterdir() if d.is_dir()), reverse=True):
+            logger.info('loading external layer from %s', layerdir)
+            if not pathlib.Path(layerdir, 'boot.yaml').exists():
```

Comment:
we should add a test which asserts this warning is logged or add a `#pragma: no cover` to this statement

Reply:
ok